config/initializers/geocoder.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 Geocoder.configure(
```

Comment:
This change breaks the current application, because the Geocoder cannot use the Google credentials anymore. Can you revert this please?

Reply:
This was wrong information, so ignore it